src/components/Button/Button.js

```diff
@@ -66,7 +66,7 @@ const Button = React.forwardRef((props, ref) => {
       {loading && (
         <Loader size="small" className={styles[`${baseClass}__loader`]} />
       )}
-      {children}
+      <div className={styles[`${baseClass}__content`]}>{children}</div>
```

Comment:
woaaah :D that will be a big change 
I saw that many components libraries did it like this and it's acceptable. Can you use `span` instead of `div`? It seems more correct.